src/vite-env.d.ts

```diff
@@ -2,3 +2,4 @@
 
 declare module '@agoric/web-components/react';
 declare module '@agoric/notifier';
+declare module '@agoric/casting';
```

Comment:
it's a shame we don't have explicit type exports yet. consider declaring some members here just so they have names. e.g. ,

```suggestion
declare module '@agoric/casting' {
  export type Leader: any,
}
```
Then your method signature can say `leader: Leader` instead of `leader: any`, which means when the types do get correctly defined we won't have to go back and figure out what all the anys are.

Reply:
Done. Yea, I don't want to get too deep into declaring all these types twice since it doesn't help if the package types change and we'll hopefully get it for free at some point.